Assets/BossRoom/Scripts/Shared/Net/ConnectionManagement/ClientGameNetPortal.cs

```diff
@@ -175,10 +174,6 @@ public void StartClient(string ipaddress, int port)
 
             switch (chosenTransport)
             {
-                case UNetTransport unetTransport:
```

Comment:
do we need switch cases for those now? Knowing UTP is our only transport?

Reply:
good point! Do we also still need the transport picker? Or should we use the same transport with Relay and direct IP?

Reply:
nuke it. if we ever need it again (which is unlikely), we'll readd it, it's not as if it was a very complex piece of tech
yagni

Reply:
shouldn't that be done in game net portal?

Reply:
I think it would be better, but we can't since it's a private field